lib/hexpm/repository/packages.ex

```diff
@@ -9,6 +9,17 @@ defmodule Hexpm.Repository.Packages do
     Repo.one!(Package.count(repositories, filter))
   end
 
+  def diff(packages, nil), do: packages
+
+  def diff(packages, remove) when is_list(remove) do
+    names = remove |> Enum.map(& &1.name)
+
+    packages
+    |> Enum.reject(&Enum.member?(names, &1.name))
```

Comment:
```suggestion
    packages
    |> Enum.reject(&(&1.name in names))
```

Reply:
Maybe `packages` should also be inlined rather than piplined?